### ISSUE TITLE
[feature] language (i18n)

### ISSUE THREAD

**Issue by 4thabang** [devguilhermy/botpress-inbox#1]
(I will add a potential issues format that you can potentially enforce as a template moving forward)
---
## Bug, Feature or Other?
- [ ] Bug
- [x] Feature
- [ ] Other

## Description:
Your application is amazing, although, it lacks proper language support. Currently I believe it is in Portuguese and a lot of people would enjoy an english variant or translation option. I believe adding other languages would be a big feat. Although, for accessibility and mass adoption, English seems like a good starting point.

## Steps to reproduce if this is a bug:
N/A

## Any other information:
N/A

**Comment by devguilhermy** [comment]
Hey @4thabang, thanks a lot the template, we shall be using it from now on!

What you are saying totally makes sense but the app is in English already 🤔 
The only thing in portuguese is the locale for date formatting, but you can change it easily in the code by just importing the preferred locale.

I actually started with PT but when I decided to make it open source I began translating all labels and comments. Did you find some untranslated text?

Anyway I think it's feasible to create a settings page for choosing language, timezone, theme and such.
Next week I might add that!

Thanks for starting this discussion!

**Comment by 4thabang** [comment]
Yeah, after digging through the code a little more I found the `ptBR` import and simply used `enUS` instead.

Although, the settings page would be a nice feature for making the app more accessible.

By the way, amazing work!

@devguilhermy

**Comment by devguilhermy** [comment]
Thanks a lot! 🙌🏽 

I will keep this open until we launch the language selection feature alright?

Besides locale you can also config the format of the date with date-fns, it's pretty neat!
Check out the docs if you don't know it already 📚 

Cheers

**Comment by 4thabang** [comment]
No worries.

Just checked it out and its really cool, thank you for pointing me to it. Will definitely be using it in future projects.

Please do let me know when the language selection feature is out
